RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/RingPublishingGDPRShowConsentScreenListener.java

```diff
@@ -0,0 +1,17 @@
+package com.ringpublishing.gdpr;
+
+public interface RingPublishingGDPRShowConsentScreenListener
```

Comment:
już się czepiałem w innym miejscu do nazwy settera, ale tu jeszcze zapytam o konwencję.

W Kotlinie prefiksuje się nazwę publicznego interfejsu także nazwą modułu, czy to wynika raczej z paczki, w której on jest dostępny?

Reply:
To wynika z konsystencji z już zrobionym modułem na iOS by mieć takie same nazwy 
https://github.com/ringpublishing/RingPublishingGDPR-iOS/tree/master/RingPublishingGDPR/Classes/Public

Dla porównania android 
https://github.com/ringpublishing/RingPublishingGDPR-Android/tree/master/RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr

Wszystkie klasy zaczynają się od RingPublishingGDPR

Chyba, że wymyślisz jedno słowo zamiast ConsentScreenListener